Integrations/integration-Ipinfo.yml

```diff
@@ -32,7 +37,7 @@ script:
     }
 
     var base = 'http://ipinfo.io/';
-    var jsonSuffix = '/json';
+    var jsonSuffix = '/json?token=' + params.token;
```

Comment:
Should it not be added to `ipinfo_field` command endpoint URL as well?

Reply:
yes, fixed